src/main/java/it/gov/pagopa/fdr/controller/middleware/filter/ResponseFilter.java

```diff
@@ -91,27 +98,27 @@ public void filter(
         throw new AppException(e, AppErrorCodeMessageEnum.ERROR);
       }
 
-      reService.sendEvent(
-          ReInterface.builder()
-              .serviceIdentifier(AppVersionEnum.FDR003)
-              .created(Instant.now())
-              .sessionId(sessionId)
-              .eventType(EventTypeEnum.INTERFACE)
-              .httpType(HttpTypeEnum.RES)
-              .httpMethod(requestMethod)
-              .httpUrl(requestPath)
-              .payload(responsePayload)
-              .header(
-                  responseContext.getHeaders().entrySet().stream()
-                      .collect(
-                          Collectors.toMap(
-                              Map.Entry::getKey,
-                              a -> Stream.of(a.getValue()).map(Object::toString).toList())))
-              .pspId(psp)
-              .fdr(fdr)
-              .organizationId(organizationId)
-              .fdrAction(fdrActionEnum)
-              .build());
+      /*reService.sendEvent(
```

Comment:
ditto

Reply:
This, as the other parts, is temporarly commented because in next commit will be redefined and maybe this structure can be re-used.